### ISSUE TITLE
`unbalanced-tuple-unpacking` false positive with `statistics.quantiles`

### ISSUE THREAD

**Issue by jamesbraza** [pylint-dev/pylint#10317]
### Bug description

```python
import statistics

q1, q2, q3 = statistics.quantiles(list(range(100)), n=4)
print(q1, q2, q3)
```

### Configuration

```ini

```

### Command used

```shell
pylint a.py
```

### Pylint output

```python
************* Module a
a.py:3:0: W0632: Possible unbalanced tuple unpacking with sequence defined at line 849 of statistics: left side has 3 labels, right side has 0 values (unbalanced-tuple-unpacking)
```

### Expected behavior

No warning from `pylint` here

### Pylint version

```shell
pylint 3.3.5
astroid 3.3.9
Python 3.12.8 (main, Jan 28 2025, 10:06:03) [Clang 16.0.0 (clang-1600.0.26.4)]
```

### OS / Environment

macOS Sequoia version 15.3.2

### Additional dependencies

```python

```

**Comment by Pierre-Sassoulas** [comment]
Fixed in pylint's [a0e601d03](https://github.com/pylint-dev/pylint/commit/a0e601d039c6ae0884ce3c52fff77d41d750cf46) (astroid's [3db2bd943](https://github.com/pylint-dev/astroid/commit/3db2bd943fdc16439d94b8634007eaf78c075536) )
